soc/xtensa/intel_adsp/ace/multiprocessing.c

```diff
@@ -89,8 +89,14 @@ void soc_mp_init(void)
 	/* Set the core 0 active */
 	soc_cpus_active[0] = true;
 #if CONFIG_SOC_INTEL_ACE15_MTPM
+#if defined(CONFIG_SMP) && (CONFIG_MP_MAX_NUM_CPUS > 1)
```

Comment:
This interesting. I think we have a few places in SOF codebase where we make this same incorrect assumption about Zephyr builds.

Reply:
Yeah. I have a branch making `SMP` depending on `CONFIG_MP_MAX_NUM_CPUS > 1`.

intel_adsp targets had some build issues when smp is turned offl.